src/renderer/src/lib/locale/en/errors.json

```diff
@@ -3,22 +3,24 @@
 	"appNameExistsError": "An app with the same name is already installed. Please choose a different name",
 	"appStoreClientNotInitializedError": "AppStoreClient not initialized",
 	"checkInitializedKeystoreError": "Error checking initialized keystore",
-	"duplicatePubKeyError": "An app with the same public key is already installed. This is not allowed due to security reasons.",
 	"devHubClientNotInitializedError": "DevHubClient not initialized",
+	"duplicatePubKeyError": "An app with the same public key is already installed. This is not allowed due to security reasons.",
+	"failedForAllAvailableHosts": "Downloading the app failed for all available peers. This typically happens due to none of the online peers having the app available themselves. Retrying again later can resolve this if new peers come online.",
```

Comment:
```suggestion
	"failedForAllAvailableHosts": "Downloading the app from any of the available peers failed. This typically happens because none of the online peers have the app available themselves. Retrying again later can resolve this if new peers come online.",
```

Reply:
https://github.com/holochain/launcher/pull/248/commits/7fdf169a9ec9f4d842867fedeb0fc4025a6aaa6a